contracts/CrossChain.sol

```diff
@@ -252,7 +321,7 @@ contract CrossChain is System, ICrossChain, IParamSubscriber{
     IRelayerIncentivize(INCENTIVIZE_ADDR).addReward(headerRelayer, msg.sender, relayFee, isRelayRewardFromSystemReward[channelIdLocal] || packageType != SYN_PACKAGE);
   }
 
-  function sendPackage(uint64 packageSequence, uint8 channelId, bytes memory payload) internal {
+  function sendPackage(uint64 packageSequence, uint8 channelId, bytes memory payload) internal whenNotSuspended {
```

Comment:
This is internal function, is it enough that we cover the external function with `whenNotSuspended `?

Reply:
fixed